src/main/java/mate/academy/spring/boot/model/Book.java

```diff
@@ -0,0 +1,33 @@
+package mate.academy.spring.boot.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data
+@Entity
```

Comment:
it's not advisable to use @Data with entities, they can have side effects, if you don't implement equals and hashcode by yourself